pkgs/development/python-modules/json2html/default.nix

```
@@ -0,0 +1,33 @@
+{
+  buildPythonPackage,
+  lib,
+  python3,
+  fetchFromGitHub,
```

```suggestion
  buildPythonPackage,
  fetchFromGitHub,
  lib,
  python3,
```
IMO it's nicer if alphabetical order is kept.